src/scss/_forms.scss

```diff
@@ -32,87 +32,34 @@ label {
 // Shared
 
 .form-check {
-  padding-left: 0;
-}
-
-.form-check-inline .form-check-input {
-  position: absolute;
+  display: flex;
+  align-items: center;
+  padding: 0;
+  .form-check-input {
+    float: unset;
+    margin-right: $spacer;
+    margin-left: 0;
+    margin-top: 0;
+  }
 }
 
 .form-check-label {
   position: relative;
-  display: flex;
-  align-items: center;
   color: $gray-600;
   font-weight: $font-weight-light;
   line-height: 1;
 
-  &::before,
-  &::after {
```

Comment:
No more before and after. We can easily change the look of them with the variables